seatunnel-connectors-v2/connector-console/src/test/java/org/apache/seatunnel/connectors/seatunnel/console/sink/ConsoleSinkWriterIT.java

```diff
@@ -0,0 +1,106 @@
+import java.lang.reflect.Method;
+import java.util.HashMap;
+import java.util.Optional;
+
+public class ConsoleSinkWriterIT {
+
+    private ConsoleSinkWriter consoleSinkWriter;
+
+    @BeforeEach
+    void setUp() {
+        String[] fieldNames = {};
+        SeaTunnelDataType<?>[] fieldTypes = {};
+        SeaTunnelRowType seaTunnelRowType = new SeaTunnelRowType(fieldNames, fieldTypes);
+        consoleSinkWriter = new ConsoleSinkWriter(seaTunnelRowType);
+    }
+
+    private Object fieldToStringTest(SeaTunnelDataType<?> dataType, Object value) {
+        Optional<Method> fieldToString = ReflectionUtils.getDeclaredMethod(ConsoleSinkWriter.class, "fieldToString", SeaTunnelDataType.class, Object.class);
+        Method method = fieldToString.orElseThrow(() -> new RuntimeException("method fieldToString not found"));
+        try {
+            return method.invoke(consoleSinkWriter, dataType, value);
+        } catch (IllegalAccessException | InvocationTargetException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    @Test
+    void arrayIntTest() {
+        Assertions.assertDoesNotThrow(() -> {
+            Integer[] integerArr = {1};
+            Object integerArrString = fieldToStringTest(ArrayType.INT_ARRAY_TYPE, integerArr);
+            Assertions.assertEquals(integerArrString, "[1]");
+            int[] intArr = {1, 2};
+            Object intArrString = fieldToStringTest(ArrayType.INT_ARRAY_TYPE, intArr);
+            Assertions.assertEquals(intArrString, "[1, 2]");
+        });
+    }
+
+    @Test
+    void stringTest() {
+        Assertions.assertDoesNotThrow(() -> {
+            String str = RandomStringUtils.randomAlphanumeric(10);
+            Object obj = fieldToStringTest(BasicType.STRING_TYPE, str);
+            Assertions.assertTrue(obj instanceof String);
+            Assertions.assertEquals(10, ((String) obj).length());
+        });
+    }
+
+    @Test
+    void hashMapTest() {
+        Assertions.assertDoesNotThrow(() -> {
+            HashMap<Object, Object> map = new HashMap<>();
+            map.put("key", "value");
+            MapType<String, String> mapType = new MapType<>(BasicType.STRING_TYPE, BasicType.STRING_TYPE);
+            Object mapString = fieldToStringTest(mapType, map);
+            Assertions.assertNotNull(mapString);
+        });
+    }
+
+    @Test
+    void rowTypeTest() {
+        Assertions.assertDoesNotThrow(() -> {
+            String[] fieldNames = {"c_byte", "c_array", "bytes"};
+            SeaTunnelDataType<?>[] fieldTypes = {BasicType.BYTE_TYPE, ArrayType.BYTE_ARRAY_TYPE, PrimitiveByteArrayType.INSTANCE};
+            SeaTunnelRowType seaTunnelRowType = new SeaTunnelRowType(fieldNames, fieldTypes);
+            byte[] bytes = RandomUtils.nextBytes(10);
+            Object[] rowData = {(byte) 1, bytes, bytes};
+            Object rowString = fieldToStringTest(seaTunnelRowType, rowData);
+            Assertions.assertNotNull(rowString);
```

Comment:
Please use equals to make sure your `fieldToString` convert are right.

Reply:
done